scripts/generate_share_images/index.ts

```diff
@@ -64,7 +70,7 @@ const BLACKLISTED_COUNTIES = [
   // Homepage share image.
   screenshots.push({
     url: '/',
-    filename: 'home',
+    filename: getHomeSocialImageFilename(),
```

Comment:
Sorrrry...  I realized this is actually the wrong place.  We don't use this script anymore since we generate the images on-demand from our firebase functions code.  We should probably go ahead and delete the script at this point, since it's no longer used.

The right place to update is here: https://github.com/covid-projections/covid-projections/blob/0672c93b5752723e9d9d89e480eabc251eaae14c/firebase/functions/src/dynamic_web_content.ts#L33

And since that runs dynamically on-demand, as long as we make that `if` statement flexible enough, it should be fine to put the date in the filename.

Reply:
No problem! 😂  - I will create a new PR and update the right place, thanks!